setup/kubernetes/rke.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Rancher Kubernetes Engine
+description: Learn how to setup a Rancher cluster for your Coder deployment
+---
+
+This deployment guide shows you how to set up a bare-metal Rancher Kubernetes
+Engine (RKE) cluster on which Coder can deploy.
+
+## Prerequisites
+
+You must have at least one Linux host (node) with the following utilities
+installed:
```

Comment:
We should link the Rancher docs on requirements:

https://rancher.com/docs/rke/latest/en/os/